### ISSUE TITLE
[CONTENT] Oscar specs and facility stateme

### ISSUE THREAD

**Issue by tserre** [brown-ccv/ccv-website-hugo#242]
Hi,
PIs need better facilities statement for oscar.

1) the current facility statement is impossible to find. if you search for "facility statement" on the CCV website it does not show up. https://ccv.brown.edu/facilities/#facilities-statement

2) the statement is too vague. Many times people need the specifics of the hardware. Specifically what are the specs of the GPU nodes, how much memory, etc.

3) there is no information about the individual condos? Carney etc

This is a big issue and it should receive a high priority... Thanks

**Comment by fernandogelin** [comment]
The search is powered by Google. If you search for `facilities`, or `facilities statement` you get results. I guess it doesn't search for similar words. 
The facilities menu is on the main navigation.

**Comment by tserre** [comment]
Thanks for the info but I have gotten multiple emails from colleagues asking me for the info and to be honest I could not find it myself and I had to email Linnea. IMHO it should be easy to find it from multiple locations (facilities is not the first thing that would come to mind even for a facility or equipment section of a grant) and it should be easily searchable (using singular or plural).

--T
--------------------------------------------------------------------------------------------
Thomas Serre | Brown University | Carney Institute for Brain Science
Associate Professor, Cognitive Linguistic & Psych. Sciences
Director, Center for Computation & Visualization

Associate Director, Center for Computational Brain Science
ANR-3IA Artificial and Natural Intelligence Toulouse Institute

T: +1 401-484-0750 | Hangout: tserre | Skype: thomas.serre | serre-lab.clps.brown.edu

( http://serre-lab.clps.brown.edu )

**Comment by fernandogelin** [comment]
@tserre I completely agree that "Facilities" is not the first thing that comes to mind. And I think that's the main issue here. When I read "Facilities" I would expect to see a list of facilities or something similar. And that's not what that section is. So people looking for something to put on a grant could easily ignore that link. I think that whole section needs to be overhauled with better guidance to researchers and potentially list all facilities and equipment CCV maintains. 
And if "Facility and Equipment" is a section of a grant, that should be in there somewhere spelled out.

**Comment by tserre** [comment]
👍

--T
--------------------------------------------------------------------------------------------
Thomas Serre | Brown University | Carney Institute for Brain Science
Associate Professor, Cognitive Linguistic & Psych. Sciences
Director, Center for Computation & Visualization

Associate Director, Center for Computational Brain Science
ANR-3IA Artificial and Natural Intelligence Toulouse Institute

T: +1 401-484-0750 | Hangout: tserre | Skype: thomas.serre | serre-lab.clps.brown.edu

( http://serre-lab.clps.brown.edu )
